Documentation/security/policy/language.rst

```diff
@@ -719,11 +719,16 @@ which is defined as follows:
 
         // PortProtocol specifies an L4 port with an optional transport protocol
         type PortProtocol struct {
-                // Port is an L4 port number. For now the string will be strictly
-                // parsed as a single uint16. In the future, this field may support
-                // ranges in the form "1024-2048"
+                // Port can be an L4 port number, or a name in the form of "http"
+                // or "http-8080". EndPort is ignored if Port is a named port.
```

Comment:
Non blocking question:  Do we really want to try to maintain a mirror of this k8s API struct in docs, feels like it will be easy for this to get out of date easily.

Reply:
I guess it's a balance. We don't change the API super often. Those structs are ultimately the source of truth for how the API behaves though. Perhaps in an ideal world we'd have docs tooling to automatically embed the latest API struct definitions in here rather than manually updating them.

Other than that, we can either copy the text over into the docs or we can copy the text plus definitions over, either way we have the question of how to make sure the docs describe the semantics of the API. I don't know if it makes a significant difference between the two.